app/src/main/java/com/example/farmeraid/farm/FarmViewModel.kt

```diff
@@ -29,7 +30,8 @@ class FarmViewModel @Inject constructor(
     private val speechRecognizerUtility: SpeechRecognizerUtility,
     private val kontinuousSpeechRecognizer: KontinuousSpeechRecognizer,
     private val appNavigator: AppNavigator,
-): ViewModel() {
+    private val userRepository: UserRepository,
```

Comment:
We shouldn't need to inject this here anymore since the user check is being done in the Repositories. Far mode doesn't need to know about it.